.github/workflows/perf_nightly.yml

```diff
@@ -0,0 +1,165 @@
+name: Nightly Benchmarks on Nyrkiö Runners (stability)
+
+on:
+  workflow_dispatch:
+    branches: ["main", "notmain", "master"]
+  schedule:
+    - cron: '24 6,11,14,20 * * *'
```

Comment:
I expect we can afford to run this more than once per 24 hours. Can we still call it nightly, or does someone have a more accurate naming in mind?